scripts/ci/pre_commit/capture_airflowctl_help.py

```diff
@@ -56,22 +56,22 @@
 def get_airflowctl_command_hash_dict(commands):
     hash_dict = {}
     for command in commands:
-        help_text = os.popen(f"python {AIRFLOW_CTL_SOURCES_PATH}/airflowctl/__main__.py {command} -h").read()
+        help_text = os.popen(f"airflowctl {command} -h").read()
```

Comment:
Hmm. Is that going to work ? I think you need to have airflowctl installed in oder to be able to run help, and from pre-commit it is not really guaranteed ?

Reply:
It couldn't find airflowctl for imports when calling `__main__`. That's why I thought if I move this to CI image built need `airflowctl` would be there. But seems not to let me convert this and the CI part back and find better way. This won't work indeed

Reply:
My idea was to make something `generate-openapi-spec` since even calling the __main__ calls the airflowctl package from the file. If I move the script to in_container and run this in a wrapper. Maybe that could solve it

Reply:
What do you think about this? This is more stable, and it should be without any errors
cc: @potiuk when you have time

Reply:
Yeah . That would work I think ..

Reply:
I tested this one and it worked fine #50987. Only updated the image and hash of the image. I see that for example, it didn't updated the hash in there normally

Reply:
Another option. You could run "uv run --isolated airflowctl --help" in the "airflow-ctl" working directory.

Reply:
Avoiding running image is good.